README.md

```diff
@@ -1 +1,13 @@
-# microverse.github.io
\ No newline at end of file
+# microverse.github.io
+
+> This is website that used for Microverse
+
+**Built With**
+Html
+Css-3
+Linters
```

Comment:
- kindly Add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) as required in the project but customize it to your project. as it was required [here](https://github.com/asadmisrani/microverse.github.io/pull/1/files#r1066289861) by the previous reviewer :+1: